openstef_dbc/utils.py

```diff
@@ -57,3 +58,21 @@ def parse_influx_result(
     result.index.name = "datetime"
     result.columns.name = ""
     return result
+
+
+def round_down_single_time_diff(time_diff, options):
+    """Rounds a time difference to the first smaller option.
+    Used to calculate the tAhead for forecasts"""
+
+    next_smaller_option = next(
+        (opt for opt in np.sort(options)[::-1] if opt < time_diff), None
```

Comment:
You could consider moving the sort and reverse of `options` into the loop function to improve performance. However the array sizes here are going to be very small so in practice this won't really matter.

Reply:
Good idea. I'll move it out of the for-loop